### DISCUSSION TITLE
Label for marker

### DISCUSSION THREAD

**Discussion by karel-zabloudil-3pg** [mapsforge/vtm#1151] [Q&A] [answered]
We have thousands markers. Is it possible to draw marker with unique label other than in the bitmap ? That would mean we have to have a separate bitmap for each marker which would consume lot of memory. If this is the only case, could you please suggest how to implement that efficiently to save the memory ? Thank you!

**Comment by devemux86** [comment, marked as answer]
You can create a `ItemizedLayer` with the common symbol, see the [MarkerOverlayActivity](https://github.com/mapsforge/vtm/blob/master/vtm-android-example/src/org/oscim/android/test/MarkerOverlayActivity.java) example.

To use different symbols need to set them on each marker, see the [AtlasMultiTextureActivity](https://github.com/mapsforge/vtm/blob/master/vtm-android-example/src/org/oscim/android/test/AtlasMultiTextureActivity.java) example.
